src/__tests__/style-component-test.js

```diff
@@ -50,6 +50,22 @@ describe('<Style> component', () => {
     `);
   });
 
+  it('add all the prefixes when we ask for', () => {
+    const output = TestUtils.renderIntoDocument(
+      <Style
+        radiumConfig={{userAgent: 'all'}}
+        rules={{div: {transform: 'rotate(90)'}}}
+      />
+    );
+
+    const style = getElement(output, 'style');
+    expectCSS(style, `
+      div {
+        -ms-transform: rotate(90);
```

Comment:
Indeed, I had no way to know the output order of the prefixes with the Symbol fail. Fixed!